forest/cli/src/cli/attach_cmd.rs

```diff
@@ -236,6 +239,42 @@ async fn send_message(
     mpool_push_message((json_message, None), auth_token).await
 }
 
+type SleepParams = (u64,);
+type SleepResult = ();
+
+async fn sleep(
+    params: SleepParams,
+    _auth_token: &Option<String>,
+) -> Result<SleepResult, jsonrpc_v2::Error> {
+    let secs = params.0;
+    time::sleep(time::Duration::from_secs(secs)).await;
+    Ok(())
+}
+
+type SleepTipsetsParams = (ChainEpoch,);
+type SleepTipsetsResult = ();
+
+async fn sleep_tipsets(
+    params: SleepTipsetsParams,
+    auth_token: &Option<String>,
+) -> Result<SleepTipsetsResult, jsonrpc_v2::Error> {
+    let mut epoch = None;
+    loop {
+        let state = sync_status((), auth_token).await?;
+        if state.active_syncs[0].stage() == SyncStage::Complete {
```

Comment:
can `[0]` panic?

Reply:
No. By construction the vector is constructed with one element.

https://github.com/ChainSafe/forest/blob/main/node/rpc/src/sync_api.rs#L56

Reply:
:+1: